Library/Formula/theora.rb

```diff
@@ -1,8 +1,9 @@
 class Theora < Formula
   desc "Open video compression format"
-  homepage "http://www.theora.org/"
+  homepage "https://www.theora.org/"
   url "http://downloads.xiph.org/releases/theora/libtheora-1.1.1.tar.bz2"
   sha256 "b6ae1ee2fa3d42ac489287d3ec34c5885730b1296f0801ae577a35193d3affbc"
+  head "https://git.xiph.org/theora.git"
```

Comment:
If you want to add a `head` line you'll need to put it in a `head do` block with autotool deps and then add a `system "./autogen.sh if build.head?` line to `def install`. There's no pre-generated `configure` script in the upstream repo.